acrobat/scripts/scripts.js

```diff
@@ -460,6 +464,15 @@ const { ietf } = getLocale(locales);
 
   setConfig({ ...CONFIG, miloLibs });
 
+  window.addEventListener('IMS:Ready', async () => {
+    const susiSignUpElems = document.querySelectorAll('a[href*="susi-sign-up"]');
+    const susiSignInElems = document.querySelectorAll('a[href*="susi-sign-in"]');
+    if (susiSignUpElems.length > 0 || susiSignInElems.length > 0) {
```

Comment:
Let's make this more generic like you did above. Something like this:
```
const susiElems = document.querySelectorAll('a[href*="susi-"]')
```
then,  in `acrobat/scripts/susiAuthHandler.js` we can determine whether it's `-sign-up` or `-sign-in` and run the correct flow.

Reply:
resolved: https://github.com/adobecom/dc/pull/819/commits/98edddaaf1edd26392871a6e961d3625089cf653